src/pip/_internal/commands/check.py

```diff
@@ -19,10 +19,26 @@ class CheckCommand(Command):
     usage = """
       %prog [options]"""
 
+    def add_options(self) -> None:
+        self.cmd_opts.add_option(
+            "--ignore-packages",
+            action="append",
```

Comment:
Should it be comma separated list or this is also fine?

Reply:
So, as it stands, would it be something like:
```
pip check --ignore-packages ninja --ignore-packages backports.cached-property
```
vs comma delimited:
```
pip check --ignore-packages ninja,backports.cached-property
```

I guess the advantage to the latter might be if there _was_ an environment variable, it might be like `PIP_CHECK_IGNORE_PACKAGES`, and would be natural as a delimited list.

Reply:
All pip options automatically generate corresponding environment variables